cluster/gce/gci/configure-helper.sh

```diff
@@ -1195,8 +1195,10 @@ function start-kube-addons {
     setup-addon-manifests "addons" "dns"
     local -r dns_controller_file="${dst_dir}/dns/kubedns-controller.yaml"
     local -r dns_svc_file="${dst_dir}/dns/kubedns-svc.yaml"
+    local -r dns_sa_file="${dst_dir}/dns/kubedns-sa.yaml"
     mv "${dst_dir}/dns/kubedns-controller.yaml.in" "${dns_controller_file}"
     mv "${dst_dir}/dns/kubedns-svc.yaml.in" "${dns_svc_file}"
+    mv "${dst_dir}/dns/kubedns-sa.yaml" "${dns_sa_file}"
```

Comment:
I think you are moving "${dst_dir}/dns/kubedns-sa.yaml" into "${dst_dir}/dns/kubedns-sa.yaml", which are the same file, so it broke.